src/TLuaInterpreter.cpp

```diff
@@ -2190,19 +2190,21 @@ int TLuaInterpreter::disableTimer( lua_State *L )
 
 int TLuaInterpreter::enableKey( lua_State *L )
 {
-    string luaSendText="";
+    string luaNameID = "";
```

Comment:
I believe this should be a `QString` by now as we pass it as one of those to `host.getKeyUnit()->enableKey(`...`)` further down anyhow.

Also there is no need to have the lua prefix for the variable name (which has been derived from the generic `luaText` that has been mindlessly copied and pasted between a whole load of functions) - `keyName` is what I would have used - but that is just my feeling on the subject! 🙊